book/pixi.md

```diff
@@ -191,7 +191,7 @@ This will result in the following manifest file:
 ```{code} toml
 :filename: pixi.toml
 :linenos:
-:emphasize-lines: 11
+:emphasize-lines: 11-12
```

Comment:
Ah, this is great as I now have learned that even though the docs say "comma separated" they also later say

> [Emphasize particular lines (comma-separated numbers which can include ranges), e.g. “3,5,7-9”](https://mystmd.org/guide/directives#directive-code-emphasize-lines).

Being able to do ranges with `-` is nice :+1:and I'll cleanup the overuse of commas I have in other exercises later.